src/execute-pr-actions.ts

```diff
@@ -106,6 +111,32 @@ function getMutationsForCommentRemovals(actions: Actions, botComments: ParsedCom
     });
 }
 
+function getMutationsForSuggestions(actions: Actions, pr: PR_repository_pullRequest) {
+    if (pr.reviews?.nodes?.find(review => review?.author?.login === "typescript-bot")) return [];
+    const mutations = Object.entries(actions.suggestions).map(([path, { startLine, line, suggestion }]) =>
+        createMutation(addPullRequestReviewThread, {
+            input: {
+                pullRequestId: pr.id,
+                path,
+                startLine: startLine === line ? undefined : startLine,
+                line,
+                body: "```suggestion\n" + suggestion,
```

Comment:
The closing \`\`\` doesn't tolerate a missing `\n`. I was mostly careful to `\n`-terminate `suggestion`, but it seemed better to omit the superfluous \`\`\`.

Reply:
I really dislike leaving it hanging...  (GH's MD format changed a few times over the years, and also, it can end up being pumped into some other tool.)

If you're trying to avoid the hassle of adding a newline when needed, there's this regexp trick to do so in a concise way:

    body.replace(/\n?$/, "\n")

Reply:
Done :heavy_check_mark: